tasks/linter.py

```diff
@@ -137,6 +137,7 @@ def go(
     headless_mode=False,
     include_sds=False,
     only_modified_packages=False,
+    verbose=False,
```

Comment:
The `verbose` flag was not present in the invoke task, so I added. I needed to validate a few things while working on this PR 😄